### DISCUSSION TITLE
Wake on LAN not working W11

### DISCUSSION THREAD

**Discussion by Semir333** [AlexGustafsson/homebridge-wol#202] [General]
Dear Devs!

I tried all of the methoad what was mentioned on the descreptions, and also help files.
At the BIOS WOL turned on, in W11 settings Magic packet etc. turned on.
Plugin see the status of the W11 PC. That was okay. The login seens to be okay, but also nothing happend also with wake up also with shut down. What can i do now? I tried something sudo plugin for the rapsberry but also not work.

19:27:19SmarthomePC[Pinger] Attempting to ping host '192.168.0.150'
19:27:19SmarthomePC[Pinger] Result of pinging: online
19:27:20SmarthomePC[NetworkDevice] Got user-triggered state change: 0
19:27:20SmarthomePC[NetworkDevice] Shutdown cycle started
19:27:20SmarthomePC[NetworkDevice] Device went from status "Online" to "ShuttingDown"
19:27:20SmarthomePC[Pinger] Stopping pinger
19:27:20SmarthomePC[NetworkDevice] Attempting to shut down the device using "net"
19:27:29SmarthomePC[NetworkDevice] Got state request - reporting ShuttingDown
19:27:30SmarthomePC[NetworkDevice] Got user-triggered state change: 0
19:28:20SmarthomePC[NetworkDevice] Got state request - reporting ShuttingDown
19:28:38SmarthomePC[NetworkDevice] Got user-triggered state change: 1
19:28:45SmarthomePC[NetworkDevice] Got user-triggered state change: 0
19:28:51SmarthomePC[NetworkDevice] Got user-triggered state change: 1

19:29:33SmarthomePC[Pinger] Result of pinging: online
19:29:33SmarthomePC[Pinger] Got 1 positives out of 1
19:29:33SmarthomePC[NetworkDevice] Got state change from pinger: online
19:29:33SmarthomePC[NetworkDevice] The device is now accessible - assuming online
19:29:33SmarthomePC[NetworkDevice] Device went from status "Offline" to "Online"

19:29:35SmarthomePC[Pinger] Attempting to ping host '192.168.0.150'
19:29:35SmarthomePC[Pinger] Result of pinging: online
19:29:37SmarthomePC[Pinger] Attempting to ping host '192.168.0.150'
19:29:37SmarthomePC[Pinger] Result of pinging: online

19:29:43SmarthomePC[Pinger] Attempting to ping host '192.168.0.150'
19:29:43SmarthomePC[Pinger] Result of pinging: online
19:29:43SmarthomePC[Pinger] Got 5 positives out of 5
19:29:43SmarthomePC[NetworkDevice] Got state change from pinger: online
19:29:45SmarthomePC[Pinger] Attempting to ping host '192.168.0.150'
19:29:45SmarthomePC[Pinger] Result of pinging: online
19:29:47SmarthomePC[NetworkDevice] Got user-triggered state change: 0
19:29:47SmarthomePC[NetworkDevice] Shutdown cycle started
19:29:47SmarthomePC[NetworkDevice] Device went from status "Online" to "ShuttingDown"
19:29:47SmarthomePC[Pinger] Stopping pinger
19:29:47SmarthomePC[NetworkDevice] Attempting to shut down the device using "net"
19:29:50SmarthomeERROR[Smarthome 6132@On] The write handler for the characteristic 'On' on the accessory 'PC' was slow to respond!
19:29:56SmarthomeERROR[Smarthome 6132@On] The write handler for the characteristic 'On' on the accessory 'PC' didn't respond at all!. Please check that you properly call the callback!

Can you send any advice for a simple W11 PC device? I have a log in with email, but it's seens to be fine, bot nothing happend.
Thank you!

**Comment by AlexGustafsson** [comment]
I've never used the plugin myself with Windows 11, so I don't know if there's anything special you need to consider. I'd recommend you to do the following:

1. If your computer is a laptop or prebuilt desktop, try to search for the name of the computer and "wake on lan" on google to see if there's anything special you need to do. If you've built the computer yourself, try to search for the name of the motherboard instead. There are many ways of trying WoL out, for example, if you have a raspberry pi you can probably find some way of sending wake on lan packages from it to your PC to try out if it works.

2. As for shutdown, the typical recommendation is to first try to get it to work outside of Homebridge. That is, try to find a command you can run on your raspberry pi to get your pc to shut down. In your case it's most likely going to be some form of SSH command, examples of which are scattered in this repository (README, wiki and old issues). If it works in the terminal, it should work with this plugin.

Splitting your issue up into these two, you can probably find better help elsewhere on forums such as https://superuser.com.

If you solve it, please share your solution to help others :)

**Reply by Semir333** [reply]
I tried a lot of methoad, but not solved yet. :/  Ping+Status work, commands are not.
